modules/apps/document-library/document-library-service/src/test/java/com/liferay/document/library/internal/util/MimeTypeSizeLimitUtilTest.java

```diff
@@ -82,4 +97,6 @@ public void testParseWildcardMimeType() {
 			(mimeType, sizeLimit) -> Assert.assertEquals("type/*", mimeType));
 	}
 
+	private static final String _PATTERN_SPECIAL_CHARACTERS = "$!#&.,;=^_+-";
```

Comment:
Optional change, replace with:

```java
	private static final char[] _PATTERN_SPECIAL_CHARACTERS = {
		CharPool.AMPERSAND, CharPool.CARET, CharPool.COMMA, CharPool.DOLLAR, 
		CharPool.EQUAL, CharPool.EXCLAMATION, CharPool.MINUS, CharPool.PERIOD,
		CharPool.PLUS, CharPool.POUND, CharPool.SEMICOLON, CharPool.UNDERLINE
	};
```

Reply:
Also, it could be named just `_SPECIAL_CHARACTERS`.

Reply:
Tomorrow I'll resend

Reply:
I'll rename the constant, then.